### PR TITLE
Blog post of Inaugural OPA winners

### PR DESCRIPTION
**Pull request by econchick**
~~There are still some TODOs - but this is an initial draft to get input on ~~

Ready for review!

### CODE REVIEW

File: www/_posts/2024/2024-05-28-outstanding-pylady-winners.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+tags: [PyLadies, Global, Community, Outreach]
+author: Lynn Root
+author_link: https://mastodon.online/@roguelynn
+---
+
+<TODO: photo of award winners at PyCon>
+
+I am sure many of us know and have experienced ourselves: the PyLadies community is made up of many
+remarkable members and volunteers that make significant contributions to bettering the Python
+community.
```
Comment (trallard) [inline www/_posts/2024/2024-05-28-outstanding-pylady-winners.md:13]:
```suggestion
The PyLadies community is made up of many
remarkable members and volunteers who make significant contributions to bettering the Python
community.
```

The intro felt a bit out of place

File: www/_posts/2024/2024-05-28-outstanding-pylady-winners.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+To celebrate that, [Cheuk Ting Ho] proposed an idea to the PyLadies Global Council last year to 
+recognize remarkable PyLadies. Cheuk saw many folks are doing amazing things for the community
+and proposed the [Outstanding PyLady Award][blog announcement]. This new Outstanding PyLady Award
+is meant to elevate diverse Pythonistas doing extraordinary community-elevating things, to highlight
+their achievements, and to thank them for all the work they've contributed.
+
```
Comment (trallard) [inline www/_posts/2024/2024-05-28-outstanding-pylady-winners.md:20]:
```suggestion
To celebrate that, [Cheuk Ting Ho] saw many folks doing amazing things for the community and proposed establishing the [Outstanding PyLady Award][blog announcement] to the PyLadies Global Council. This new Outstanding PyLady Award
is meant to elevate diverse Pythonistas doing extraordinary community-elevating things, highlight
their achievements, and to thank them for all the work they've contributed.

```

File: www/_posts/2024/2024-05-28-outstanding-pylady-winners.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+Each year, the PyLadies Global Council will be selecting a handful of folks from nominations from
```
Comment (trallard) [inline www/_posts/2024/2024-05-28-outstanding-pylady-winners.md:21]:
```suggestion
Moving forward, every year the PyLadies Global Council will be selecting a handful of folks from nominations from
```

File: www/_posts/2024/2024-05-28-outstanding-pylady-winners.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+the Python community at large (anyone in the Python community is encouraged to 
+[nominate][nomination form]!). Award winners are recognized at PyCon US, and receive their choice of
+either free registration & travel assistance to PyCon US, or a cash award of $1,000. Its intention
+is to demonstrate that their service and contributions to the PyLadies community does lead to
+recognition and reward. They also get a cute little pin to wear on their lanyards for all PyCons to
+come, if they wish. <TODO: image of pin>
+
+This is the first year of the Outstanding PyLady awards, and we got a whopping _**84 submissions**_ 
+for nominations. Unfortunately, we do not have the capacity to award all the nominations, although 
+all of them are very clearly deserving.
```
Comment (trallard) [inline www/_posts/2024/2024-05-28-outstanding-pylady-winners.md:31]:
```suggestion
This is the first year of the Outstanding PyLady awards, and we got a whopping _**84 submissions**_ 
for nominations. Unfortunately, we do not have the capacity to award all the nominees, although 
all of them clearly deserve it. 
```

**Review by trallard — COMMENTED** [review]
Added a bunch of suggestions but looking good so far

**Review by willingc — DISMISSED** [review]

**Review by BethanyG — DISMISSED** [review]
I am getting `Unable to render rich display` errors for the following files:

`www/_assets/images/blog/opa_badge_pin.png` & `www/_assets/images/blog/opa_winners_auction_2024.png`

Beyond that, this looks good!  😄

**Comment by willingc** [comment]
> I am getting `Unable to render rich display` errors for the following files:
> 
> `www/_assets/images/blog/opa_badge_pin.png` & `www/_assets/images/blog/opa_winners_auction_2024.png`
> 
> Beyond that, this looks good! 😄

Weird latency but they did eventually load.

**Comment by mahya8585** [comment]
LGTM🥰 Thank you for creating the article!

File: www/_posts/2024/2024-05-28-outstanding-pylady-winners.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+tags: [Global, "Outstanding PyLady Award"]
+author: Lynn Root
+author_link: https://mastodon.online/@roguelynn
+---
+
+Announcing the first recipients of the Outstanding PyLady Awards (🎉 OPA 🎉)!
+
+<figure>
+    <img src="{{ get_asset('images/blog/opa_winners_2024.png') }}" alt="Award winners posing at PyCon US 2024 Closing Ceremony" width="70%" style="margin-left: auto; margin-right: auto; margin-bottom: 2em; margin-top: 2em; display: block;">
+<figcaption style="text-align: center;"><i>Abigail Mesrenyame Dogbe (left-most), Maaya Ishida (second from right), and Jessica Green (right-most) posing with their blue gem awards with Lynn Root (second from left) at the closing ceremony of PyCon 2024. Photo taken by <a href="https://x.com/psobot">@psobot</a>.</i></figcaption></figure>
```
Comment (berlin-pyladies) [inline www/_posts/2024/2024-05-28-outstanding-pylady-winners.md:13]:
```suggestion
<figcaption style="text-align: center;"><i>Abigail Mesrenyame Dogbe (left-most), Maaya Ishida (second from right), and Jessica Greene (right-most) posing with their blue gem awards with Lynn Root (second from left) at the closing ceremony of PyCon 2024. Photo taken by <a href="https://x.com/psobot">@psobot</a>.</i></figcaption></figure>
```

I wrote this in the email but then realised I could just add it here :)

**Pull request merged**